.circleci/config.yml

```diff
@@ -79,10 +79,35 @@ workflows:
   version: 2
   commit:
     jobs:
-      - build
+      # workflows run only on master, legacy branches, or branches beginning
+      # with prebid-.
+      - build:
+          filters:
+            branches:
+              only:
+                - master
+                - /.*legacy.*/
+                - /prebid-.*/
+            pull_requests:
```

Comment:
i do not see this syntax at https://circleci.com/docs/configuration-reference/#jobfilters, probable hallucination